Sources/Overload/OvEditor/src/OvEditor/Core/EditorActions.cpp

```diff
@@ -483,9 +483,10 @@ OvMaths::FVector3 OvEditor::Core::EditorActions::CalculateActorSpawnPoint(float
 	return sceneView.GetCameraPosition() + sceneView.GetCameraRotation() * OvMaths::FVector3::Forward * p_distanceToCamera;
 }
 
-OvCore::ECS::Actor & OvEditor::Core::EditorActions::CreateEmptyActor(bool p_focusOnCreation, OvCore::ECS::Actor* p_parent)
+OvCore::ECS::Actor & OvEditor::Core::EditorActions::CreateEmptyActor(bool p_focusOnCreation, OvCore::ECS::Actor* p_parent, const std::string& p_name)
 {
-	auto& instance = m_context.sceneManager.GetCurrentScene()->CreateActor();
+    const auto currentScene = m_context.sceneManager.GetCurrentScene();
```

Comment:
`const auto currentScene` should be `auto* const currentScene` since `m_context.sceneManager.GetCurrentScene()` returns a pointer

Reply:
`GetCurrentScene()` should be marked as const

Reply:
Since we're at it, `CalculateActorSpawnPoint(10.0f)` line 495 can also be marked `const` as well as his parameter `float p_distanceToCamera`

Reply:
When using `auto`, pointer types are automatically deduced, so `auto*` isn't required. Making `GetCurrentScene` is out of the scope of this PR.

Same thing about `CalculateActorSpawnPoint`.